README.md

```diff
@@ -52,8 +52,7 @@ https://github.com/kiwicom/eslint-plugin-kiwicom/blob/master/src/rules/index.js
 ## How to contribute
 
 * Clone repository
-* Install dependencies
-* Optionally use [Wallabyjs](https://wallabyjs.com/)
```

Comment:
why do you want to remove this? It's suggesting that the wallaby config is there.

Reply:
hmm I wonder if it's relavant. if you have wallaby then it will automatically detect that this project has the config, no?